miner/scroll_worker.go

```diff
@@ -354,8 +355,10 @@ func (w *worker) mainLoop() {
 			w.current = nil
 		}
 
+		idleStart := time.Now()
```

Comment:
> This will not count the above reorg and retry logic

Correct. Time spent on those are still "busy" time, not idle.

> new metric

Yeah, but I don't think we will have reorgs frequent enough to be significant.